docs/user/production-considerations/task-manager-troubleshooting.asciidoc

```diff
@@ -681,32 +809,133 @@ Suppose the output of `stats.workload.value` looked something like this:
+
+Suppose an alternate scenario, where you see the following output under `stats.capacity_estimation.value`:
+
+[source,json]
+--------------------------------------------------
+{
+  "observed": {
+    "observed_kibana_instances": 2, # <1>
+    "max_throughput_per_minute_per_kibana": 200,
+    "max_throughput_per_minute": 400, # <2>
+    "minutes_to_drain_overdue": 12, # <3>
+    "avg_recurring_required_throughput_per_minute": 354, # <4>
+    "avg_recurring_required_throughput_per_minute_per_kibana": 177, # <5>
+    "avg_required_throughput_per_minute": 434, # <6>
+    "avg_required_throughput_per_minute_per_kibana": 217
+  },
+  "proposed": {
+    "min_required_kibana": 2, # <7>
+    "proposed_kibana": 3, # <8>
+    "avg_recurring_required_throughput_per_minute_per_kibana": 118, # <9>
+    "avg_required_throughput_per_minute_per_kibana": 145 # <10>
+  }
+}
+--------------------------------------------------
+<1> These estimates assume that there are two {kib} instance actively executing tasks
+<2> The maximum available throughput in the system currently is 400 tasks per minute
+<3> Based on past throughput the overdue tasks in the system should be executed within 12 minute
+<4> On average the recurring tasks in the system have historically required a throughput of 354 tasks per minute
+<5> On average each {kib} instance utilizes 177 tasks per minute of its capacity to execute recurring tasks
+<6> On average the tasks in the system have historically required a throughput of 434 tasks per minute
+<7> The system estimates that at least two {kib} instances are required to run the current recurring workload
+<8> The system recommends provisioning three {kib} instances to handle the workload
+<9> Once a third {kib} instance is provisioned, the capacity utilized by each instance to execute recurring tasks should drop from 177 to 118 tasks per minute
+<10> Taking into account historical ad-hoc task execution, we estimate the throughput required of each {kib} instance will drop from 217 task per minute to 145, once a third {kib} instance is provisioned
+
+Evaluating by these estimates, we can infer some interesting attributes of our system:
+
+* These estimates are produced based on the assumption that there are two {kib} instances in the cluster. This number is based on the number of {kib} instances actively executing tasks in recent minutes. At times this number might fluctuate if {kib} instances remain idle, so validating these estimates against what you know about the system is recommended.
+* There appear to be so many overdue tasks that it would take 12 minutes of executions to catch up with that backlog. This does not take into account tasks that might become overdue during those 12 minutes, so while this congestion might be temporary, the system could also remain consistently under provisioned and might never drain the backlog entirely.
```

Comment:
```suggestion
* There appear to be so many overdue tasks that it would take 12 minutes of executions to catch up with that backlog. This does not take into account tasks that might become overdue during those 12 minutes. Although this congestion might be temporary, the system could also remain consistently under provisioned and might never drain the backlog entirely.
```